charts/linkerd2/values.yaml

```diff
@@ -369,19 +367,9 @@ policyValidator:
   # as no certificate will be generated.
   caBundle: |
 
-# -|- CPU, Memory and Ephemeral Storage resources required by the SP validator (see
-#`proxy.resources` for sub-fields)
-#spValidatorResources:
-
-# If the namespace is controlled by an external tool or can't be installed with Helm
-# you can disable its installation. In this case:
-# - The namespace created by the external tool must match the namespace value above
-# - The external tool needs to create the namespace with the label:
-#     config.linkerd.io/admission-webhooks: disabled
-
-# installNamespace -- Set to false when installing Linkerd in a custom namespace. See the
-# [Linkerd documentation](https://linkerd.io/2/tasks/install-helm#customizing-the-namespace) for more information.
-installNamespace: true
+# -- CPU, Memory and Ephemeral Storage resources required by the SP validator (see
+# `proxy.resources` for sub-fields)
+spValidatorResources:
```

Comment:
Perhaps a bit besides the point but this was commented before, do we still want to have it commented by default?

Reply:
I uncommented it so that the description is picked up in the chart's README.md file :wink: . Doing so doesn't impact the templates I think (no empty resources appear or anything like that).